### PR TITLE
Customizer updates

### PR DESCRIPTION
**Pull request by chriswallace**
There were also a number of improvements to the Sass variable structure, grunt config, dev dependencies, and other needed fixes.

### CODE REVIEW

**Comment by chriswallace** [comment]
@fjarrett Can you review this PR?

File: inc/customizer.php [hunk 1]
```diff
@@ -94,7 +94,19 @@ function primer_customize_register( $wp_customize ) {
 		'priority' => 1,
 	) );
 
-	// Add page background color setting and control.
+	// Add menu background color setting and control.
+	$wp_customize->add_setting( 'menu_background_color', array(
+		'default'           => $color_scheme[3],
```
Comment (frankiejarrett) [inline inc/customizer.php]: @chriswallace This should be `$color_scheme[2]` according to https://github.com/godaddy/wp-primer-theme/pull/19/files#diff-b8746809c04b1698add2a397c385bc9eR370

File: inc/customizer.php [hunk 2]
```diff
@@ -94,7 +94,19 @@ function primer_customize_register( $wp_customize ) {
...
+	) );
+
+	$wp_customize->add_control( new WP_Customize_Color_Control( $wp_customize, 'menu_background_color', array(
+		'label'       => __( 'Menu Background Color', 'primer' ),
+		'section'     => 'colors',
+	) ) );
+
+	// Add tagline text color setting and control.
 	$wp_customize->add_setting( 'tagline_text_color', array(
 		'default'           => $color_scheme[2],
```
Comment (frankiejarrett) [inline inc/customizer.php]: @chriswallace This should be `$color_scheme[3]`

Comment (frankiejarrett) [inline inc/customizer.php]: @chriswallace This is actually a cascading problem, all of the `$wp_customize->add_setting` methods will need to have their defaults updated to use the correct index as defined in `primer_get_color_schemes()`

Comment (chriswallace) [inline inc/customizer.php]: OK, will review and fix.

**Comment by chriswallace** [comment]
@fjarrett Please check the latest changes, that issue should be resolved. I also moved our IE compatibility files over to the parent theme as well.

**Comment by frankiejarrett** [comment]
@chriswallace Thanks for the fixes so far.

Either the Link Color and Menu Background Color should not be the same, or the actual hover state color on menu items needs to be different than the Link Color. It's causing issues in every color scheme:

screen shot 2016-07-12 at 3 40 11 pm

**Comment by frankiejarrett** [comment]
@chriswallace @schrapel I'm going to merge this because it's technically working and I'd rather not have two forks of the same feature branch. We just need to clean up some of the colors used in the schemes. Let's continue all this work in the `feature/customizer` branch on #20. Chris, you should have push access to this repo so you don't have to work off of your own fork.

**Comment by chriswallace** [comment]
@fjarrett great, thanks.

**Pull request merged**